helix-core/src/main/java/org/apache/helix/controller/rebalancer/waged/WagedRebalancer.java

```diff
@@ -255,11 +261,14 @@ private void refreshBaseline(ResourceControllerDataProvider clusterData,
       Map<HelixConstants.ChangeType, Set<String>> clusterChanges, Map<String, Resource> resourceMap,
       final CurrentStateOutput currentStateOutput) throws HelixRebalanceException {
     LOG.info("Start calculating the new best possible assignment.");
+    LatencyMetric partialRebalanceMetric = new PartialRebalanceMetric("PartialRebalanceLatency");
+    _metricCollector.addMetric(partialRebalanceMetric);
+    partialRebalanceMetric.start();
```

Comment:
I like this design. Shall we call it startTiming or another more specific method name?